yandex_contest/bin_sum.py

```diff
@@ -0,0 +1,53 @@
+def convert(string):
+    list_1 = []
+    list_1[:0] = string
+    list_x = [int(i) for i in list_1]
+    return list_x
+
+def main():
+    num_1 = input()
+    list_1 = convert(num_1)
+
+    num_2 = input()
+    list_2 = convert(num_2)
+
+    result = []
+    summa = 0
+    rest = 0
```

Comment:
It's always better to define variables as close to their usage as possible to
a) avoid potential bugs: eg, you can use them accidentally in the code in the middle, so they will have unexpected values
b) simplify reading of the code, as you see the variable and then immediately what it's used for and how